Classes/Core/ConsoleBootstrap.php

```diff
@@ -74,7 +74,7 @@ public function initialize(\Composer\Autoload\ClassLoader $classLoader)
             if (!is_callable([$this, 'setRequestType'])) {
                 $this->defineTypo3RequestTypes();
             }
-            $this->requestId = uniqid('console_request_', true);
+            $this->requestId = uniqid('console_request_', false);
```

Comment:
I'd go for `substr(md5(uniqid('console_request_', true)), 0, 13);` then, like it is in the core